packages/data/tests/unit/adapters/dimensions/keg-test.js

```diff
@@ -36,16 +37,25 @@ module('Unit | Adapters | Dimensions | Keg', function(hooks) {
   setupTest(hooks);
 
   hooks.beforeEach(function() {
-    this.owner.register('model:dimension/dimensionOne', EmberObject.extend({ name: 'dimensionOne' }));
+    this.owner.register('model:dimension/dummy.dimensionOne', EmberObject.extend({ name: 'dimensionOne' }));
+    this.owner.register('model:dimension/blockhead.dimensionFour', EmberObject.extend({ name: 'dimensionFour' }));
 
     Adapter = this.owner.lookup('adapter:dimensions/keg');
 
     Keg = Adapter.get('keg');
-    Keg.pushMany('dimension/dimensionOne', Records);
+    Keg.pushMany('dimension/dummy.dimensionOne', Records, { dataSourceName: 'dummy' });
```

Comment:
Seems strange that we need to provide the namespace in the dimension lookup and in the options hash. Is there any case where the two would be different?

Reply:
this is because the way the keg adapter works. It was decided that they concept of namespaces in the keg, and datasources separate.  Since the keg adapter makes uses of both concepts we see where we have to provide both in this situation.  It's not really a matter of they'll ever be different but more for conceptual consistency.